components/ndatasource/org.wso2.carbon.ndatasource.ui/pom.xml

```diff
@@ -34,7 +34,10 @@
                         <Bundle-Version>${project.version}</Bundle-Version>
                         <Carbon-Component>UIBundle</Carbon-Component>
                         <Export-Package>
-                            org.wso2.carbon.ndatasource.ui.*,
+                            org.wso2.carbon.ndatasource.ui,
+                            org.wso2.carbon.ndatasource.ui.config,
+                            org.wso2.carbon.ndatasource.ui.i18n,
+                            !org.wso2.carbon.ndatasource.ui.stub.*,
```

Comment:
Shall we do it like this -
org.wso2.carbon.ndatasource.ui.*,
!org.wso2.carbon.ndatasource.ui.stub.*,